test/format/format.sh

```diff
@@ -178,12 +180,12 @@ format_binary=$(find_file "t")
 	echo "$name: format program \"$format_binary\" not found"
 	exit 1
 }
-[[ $abort_test -ne 0 ]] && {
-    wt_binary=$(find_file "wt")
-    [[ ! -x "$wt_binary" ]] && {
-	echo "$name: wt program \"$wt_binary\" not found"
-	exit 1
-    }
+[[ $abort_test -ne 0 ]] || [[ $smoke_test -ne 0 ]] && {
+	wt_binary=$(find_file "wt")
+	[[ ! -x "$wt_binary" ]] && {
+		echo "$name: wt program \"$wt_binary\" not found"
+		exit 1
+	}
```

Comment:
Not sure why all this additional whitespace indenting was added here?

Reply:
I reworked the whitespace indentation in this block to match the rest of the file. I don't think we have any shell script standard formatting, so not sure if it's "right" or not.

Reply:
That's fine with me. Just wasn't sure if there was something more subtle going on that I missed.